### ISSUE TITLE
Posts: posts aren't being published at the scheduled time

### ISSUE THREAD

**Issue by catkit** [Automattic/wp-calypso#38864]
This user reported that their scheduled posts aren't published at the scheduled time and they have to manually publish them.

Their timezone is correct and their future scheduled posts are correctly set to be published at 2am both at Calypso and WP Admin.

The posts they claim were scheduled to be published at 2am shows 2am in the dashboard and in Publicize RC, but not in their site's activity logs which reflects what they described. 

Site: https://revivallifedevotionshome.wordpress.com/
Published post sample: http://revivallifedevotions.home.blog/2020/01/15/day-3/
Followup ticket: 2631805-zen

#### Context / Source

#user-report

**Comment by kriskarkoski** [comment]
I think the behavior here is expected if not super clear. Whatever time you set a post for is what will show in WP Admin even if it was published later, either due to not getting traffic or being manually publish and backdated. From my experience, Activity Log will always show the time of the action.

Did we check if they get sufficient traffic to trigger the post?

**Comment by catkit** [comment]
> Did we check if they get sufficient traffic to trigger the post?

I wasn't aware that scheduled post would be published only after someone that isn't logged actually visits the site, my bad.

Closing this now.
